explore/src/main/scala/explore/constraints/ConstraintsPanel.scala

```diff
@@ -172,7 +169,7 @@ object ConstraintsPanel {
                 constraintSetId = id.assign,
                 // programId = constraintSet.get.programId,
                 programId = "p-2",
-                name = NonEmptyString.unsafeFrom(constraintSet.get.name + " - Copy"),
+                name = NonEmptyString.unsafeFrom(s"${constraintSet.get.name} [$id]"),
```

Comment:
Do we want to show the `id` in the combobox?

Reply:
If we show the id the combobox we can keep the same name as the original when copying. Is that what you are suggesting?